### PR TITLE
SAK-45797 Announcements: Warn that Email Notifications require a published site

### PR DESCRIPTION
**Pull request by hornersa**
https://jira.sakaiproject.org/browse/SAK-45797

### CODE REVIEW

File: announcement/announcement-tool/tool/src/webapp/vm/announcement/chef_announcements-revise.vm [hunk 1]
```diff
@@ -403,6 +403,11 @@ function resizeFrame(){
 
 			#if ($notification)
 				<hr class="itemSeparator" />
+
+				#if (! $site.isPublished())
```
Comment (ern) [inline announcement/announcement-tool/tool/src/webapp/vm/announcement/chef_announcements-revise.vm:407]:
```suggestion
				#if (!$site.isPublished())
```

Comment (hornersa) [reply announcement/announcement-tool/tool/src/webapp/vm/announcement/chef_announcements-revise.vm:407]: Sounds good. I'll force push into this branch this suggestion.

File: announcement/announcement-api/api/src/bundle/announcement.properties [hunk 2]
```diff
@@ -230,6 +230,7 @@ revise.group.list.desc=Description
 ### (moot) revise.group.list.sortdescasc=Sort by group description ascending
 ### (moot) revise.group.list.sortdescdesc=Sort by group description descending
 
+revise.notify_alert = <strong>Email notifications will not be sent</strong> unless this site is published before posting the announcement; however, before clicking Publish Now above, copy your announcement text or post the announcement as hidden (in draft mode) to avoid losing your work.
```
Comment (ottenhoff) [inline announcement/announcement-api/api/src/bundle/announcement.properties:233]: Here is how I would phrase this:

<strong>Email notifications will not be sent</strong> while this site is unpublished.

Comment (hornersa) [reply announcement/announcement-api/api/src/bundle/announcement.properties:233]: That works for me. Later today I'll implement a force push to incorporate that edit. Thanks.

**Pull request merged**